app/views/layouts/_flashes.html.haml

```diff
@@ -1,3 +1,4 @@
 - flash.each do |key, value|
   - unless %w{hidden ga timedout}.include? key
-    = govuk_notification_banner(key.capitalize, value, class: "govuk-notification-banner--#{key}")
+    = govuk_notification_banner(title_text: key.capitalize, success: key == 'alert', classes: "auto-dismiss-banner") do |nb|
```

Comment:
I don't think this is working quite right - when I log in (or log out) I get the banner in blue, when previously it was green. 

![image](https://github.com/user-attachments/assets/86c2d3bb-2086-45ff-b687-9e885996147f)

I think this is because the `key` in these cases is `notice` not `alert`, so `success` is set to `false`. Changing `key == 'alert'` to `key == 'notice'` might fix this, but I'm not sure what affect it might have elsewhere...